mlflow/utils/cli_args.py

```diff
@@ -123,6 +123,23 @@ def _create_env_manager_option(help_string, default=None):
 """,
 )
 
+ENV_MANAGER_DOCKERFILE = _create_env_manager_option(
```

Comment:
We shouldn't change the default value for ENV_MANAGER as all clis are depending on it, changing its default value would cause self.env_manager to be None in lots of functions and that's also a breaking change